views/index.erb

```diff
@@ -0,0 +1,37 @@
+<!doctype html>
+<head>
+
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+  <link rel="stylesheet" type="text/css" href="/style.css">
+
+</head>
+
+<body>
+
```

Comment:
The blank lines between elements in the body makes reading the code easier. I'm not sure what the Rubocop convention is, but some of the lines at the top and at the end could be removed.